Tests/AblyChatTests/DefaultRoomsTests.swift

```diff
@@ -6,7 +6,7 @@ struct DefaultRoomsTests {
     @Test
     func get_returnsRoomWithGivenID() async throws {
         // Given: an instance of DefaultRooms
-        let realtime = MockRealtime.create()
+        let realtime = MockRealtime.create(channels: .init(channels: [.init(name: "basketball::$chat::$chatMessages")]))
```

Comment:
This is what I was mentioning in our call — before we didn't specify a channel name because it made no difference to the test; now the tests specify a channel name and it's not clear as a reader whether this name is important or not.

Reply:
ahh i get you, I've added a comment to explain why this is now necessary